### PR TITLE
OCD-3612: Add AQA test for supporting v20 template upload

### PR DESCRIPTION
**Pull request by palakpatel311**
[#OCD-3612]

### CODE REVIEW

File: e2e/components/surveillance/edit/surveillance-edit.po.js [hunk 1]
```diff
@@ -151,8 +151,8 @@ class SurveillanceEditComponent {
     return $(elements.editRequirement);
   }
 
-  get removeNonconformity () {
-    return $(elements.removeNonconformity);
+  get removenonConformity () {
```
Comment (kekey1) [inline e2e/components/surveillance/edit/surveillance-edit.po.js]:
```suggestion
  get removeNonConformity () {
```

File: e2e/components/upload/upload-listing/upload-listing.spec.js [hunk 2]
```diff
@@ -34,4 +34,9 @@ describe('When uploading a listing as ONC-ACB', () => {
     uploadListingComponent.uploadListing('../../../resources/listings/2015_v18_AQA2.csv');
     assert.include(uploadListingComponent.listingUploadText.getText(),'was uploaded successfully. 1 pending products are ready for confirmation.', 'File has uploaded successfully');
   });
+
+  it('can upload v20 template', () => {
```
Comment (kekey1) [inline e2e/components/upload/upload-listing/upload-listing.spec.js:38]: Does this also test the flexible upload component? If not, can we have tests for both?

**Review by kekey1 — CHANGES_REQUESTED** [review]

File: e2e/components/surveillance/edit/surveillance-edit.po.js (hunk 1 above)
Comment (palakpatel311) [reply e2e/components/surveillance/edit/surveillance-edit.po.js]: we changed format of noncomformity to "nonConformity" so kept the same. But this was your suggestion originally so I could change this one, too. let me know

File: e2e/components/upload/upload-listing/upload-listing.spec.js (hunk 2 above)
Comment (palakpatel311) [reply e2e/components/upload/upload-listing/upload-listing.spec.js:38]: Flexible upload also test this which is added in upload-listing-beta-dp.js

File: e2e/components/surveillance/edit/surveillance-edit.po.js (hunk 1 above)
Comment (kekey1) [reply e2e/components/surveillance/edit/surveillance-edit.po.js]: Yeah, I think though to be properly camel-cased, each word needs to start with a capital letter.

**Review by kekey1 — DISMISSED** [review]

**Review by andlar — DISMISSED** [review]

**Review by athomas-git — DISMISSED** [review]

**Review by ashwinimore — DISMISSED** [review]

**Review by andlar — APPROVED** [review]

**Review by athomas-git — APPROVED** [review]

**Review by ashwinimore — APPROVED** [review]

**Review by kekey1 — APPROVED** [review]

**Pull request merged**